### PR TITLE
updated readme with cloud-services-config setup

### PR DESCRIPTION
**Pull request by resoluteCoder**
updated the readme file to include instructions for the cloud-services-config repo and added configuration step for edge-frontend

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,31 @@ Install all dependencies
 >$ npm install
 ```
 
+Replace line 6 in `edge-frontend/profiles/local-frontend.js` to:
+
+```bash
```
[the remark this answers is not in the document]
Comment (resoluteCoder) [reply README.md]: Ah right. Thanks for pointing that out, it's fixed.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,31 @@ Install all dependencies
 >$ npm install
 ```
 
+Replace line 6 in `edge-frontend/profiles/local-frontend.js` to:
+
+```js
+const routes = {
+    '/config': { host: 'http://127.0.0.1:8889' },
```
Comment (boaz0) [inline README.md:17]: @karelhala should we put this in our local-frontend.js or the insights-proxy default routes?

**Review by boaz0 — COMMENTED** [review]
@resoluteCoder all in all LGTM. I just want @karelhala :eyes: on something to see what he thinks.

GJ :1st_place_medal:

File: README.md (hunk 2 above)
Comment (karelhala) [reply README.md:17]: This is not mandatory, once edge is on stable branches we won't be needing it anymore. Plus if you run `BETA=true npm run start:proxy` you should be able to see the UI on `https://ci.foo.redhat.com:1337/beta/edge`

**Review by karelhala — APPROVED** [review]
Looking good, I would also mention in the docs that running `BETA=true npm run start:proxy` will launch the app in BETA environment which means the UI is accessible on `https://ci.foo.redhat.com:1337/beta/edge`

**Review by boaz0 — APPROVED** [review]

**Pull request merged**